hedera-node/hedera-file-service-impl/src/main/resources/genesis/simpleFeesSchedules.json

```diff
@@ -29,7 +29,7 @@
     { "name":  "CREATED_ACCOUNTS",               "fee": 0 },
     { "name":  "CUSTOM_FEE",                     "fee": 0 },
     { "name":  "GAS",                            "fee": 1 },
-    { "name":  "ALLOWANCES",                     "fee": 2000 },
+    { "name":  "ALLOWANCES",                     "fee": 5000000 },
```

Comment:
I think in old fees, if we are adjusting allowances, we were charging less. We should confirm with Keith what the price is here. cc : @joshmarinacci

Reply:
I think he mentioned it should be the same as baseFee , 500000000